python/pycrdt/text.py

```diff
@@ -103,19 +103,30 @@ def __setitem__(self, key: int | slice, value: str) -> None:
                 del self[key]
                 self.integrated.insert(txn._txn, key, value)
             elif isinstance(key, slice):
-                start, stop = self._check_slice(key)
-                if start != stop:
-                    self.integrated.remove_range(txn._txn, start, stop)
+                start, length = self._check_slice(key)
+                if length > 0:
+                    self.integrated.remove_range(txn._txn, start, length)
                 self.integrated.insert(txn._txn, start, value)
             else:
                 raise RuntimeError(f"Index not supported: {key}")
 
+    def replace_range(self, value: str, start: int, stop: int) -> None:
```

Comment:
This method doesn't exist on a builtin `list`, and it does the same as:
```py
text[start:stop] = value
```
Do you think it is worth it?

Reply:
Please the [above](https://github.com/jupyter-server/pycrdt/pull/68#discussion_r1461774125)

Reply:
Same comment as for `remove_range`.

Reply:
Reverted.